src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt

```diff
@@ -134,3 +134,10 @@ implication        ,ARM64 ,Sha1      ,ArmBase
 implication        ,ARM64 ,Sha256    ,ArmBase
 implication        ,ARM64 ,Vector64  ,AdvSimd
 implication        ,ARM64 ,Vector128 ,AdvSimd
+
+; Known CPUs (instruction groups)         ,lower baselines included by implication
+instructionsetgroup  ,sandybridge  ,X64   ,avx pclmul popcnt
+instructionsetgroup  ,sandybridge  ,X86   ,avx pclmul popcnt
```

Comment:
Would be nice to have `copyinstructionsetgroups,X86 ,X64` so we don't need to "reiterate" the x86 vs x64 difference